### ISSUE TITLE
Trained googlenet's outputs are negative value. Why negative?

### ISSUE THREAD

**Issue by bemoregt** [soumith/imagenet-multiGPU.torch#106]
Hi, @Smerity @bobbens @pedropgusmao @soumith @ppwwyyxx 

My trained googlenet's test outputs are negative values.
----------------------------------------------------------------------------
 -5.2452e-06 -1.2193e+01 -6.6188e-01 -7.2542e-01
[torch.CudaTensor of size 1x4]

Why they are negative?
Biggest value(smallest absolute value for negative) is the winner ?

Thanks in advances ~

**Comment by mhmtsarigul** [comment]
Dear bemoregt; 

Output layer of the googlenet is logarithmic softmax layer.

Softmax layer gives outputs between 0 and 1. (exp(x_i) / exp(x).sum())
So, Outputs of LogSoftMax layer will be negative. (log( exp(x_i) / exp(x).sum() ))
